lib/greed/die.rb

```diff
@@ -0,0 +1,18 @@
+module Greed
+  class Die
+    attr_reader :value
+    attr_reader :sides
+
+    # Create a die with a specified number of sides.
+    # Uses a six sided die as default if not provided.
+    def initialize(sides=6)
+      @sides = sides
+      @value = 0
+    end
+
+    # Roll the die and get the value
+    def roll
+      @value = rand(1..@sides)
+    end
```

Comment:
This does not behave like actual dice.  Don't know if that's what you would want or not.